packages/module/src/Message/Message.tsx

```diff
@@ -109,6 +116,7 @@ export interface MessageProps extends Omit<React.HTMLProps<HTMLDivElement>, 'rol
   isLiveRegion?: boolean;
   /** Ref applied to message  */
   innerRef?: React.Ref<HTMLDivElement>;
+  tableProps?: TableProps;
```

Comment:
Can we include a prop description here? Just something simple like "Addition props passes to the table"

Reply:
Thanks! I missed this.